cli/src/command/account.rs

```diff
@@ -644,8 +664,19 @@ pub async fn node_info_command(account: &Account) -> Result<(), Error> {
 }
 
 /// `output` command
-pub async fn output_command(account: &Account, output_id: String) -> Result<(), Error> {
-    let output = account.get_output(&OutputId::from_str(&output_id)?).await;
+pub async fn output_command(account: &Account, selector: OutputSelector) -> Result<(), Error> {
+    let output = match selector {
+        OutputSelector::Id(id) => account.get_output(&id).await,
+        OutputSelector::Index(index) => {
+            let mut outputs = account.outputs(None).await?;
+            outputs.sort_by(|a, b| {
+                a.metadata
+                    .milestone_timestamp_booked()
+                    .cmp(&b.metadata.milestone_timestamp_booked())
```

Comment:
I have one last concern, when we have many outputs with the exact same milestone_timestamp_booked, which is currently my case, I don't think we have any guarantee that both functions will order the same way. I would order by (timestamp, ID) pair to be sure?

Reply:
Actually I don't know, it's unclear

Reply:
very good, Thibault, you're right